data/overview.yaml

```diff
@@ -22,6 +22,47 @@ Linear algebra:
   finite dimension representations:
     examples: ''
     Schur's lemma: ''
+  finite dimensional:
+    vector space: ''
+    canonical isomorphism with K^n: ''
+    rank of vectors: ''
+    rank of a linear map: ''
+    dual vector space: ''
+    transpose of a linear map: ''
+    dual basis: ''
+    bidual: ''
+    orthogonality: ''
+  multilinear:
+    multilinear map: ''
+    determinant of vectors: ''
+    determinant of endomorphisms: ''
+    special linear group: ''
+    orientation of a R-valued vector space: ''
+  matrices:
+    commutative ring valued matrices: ''
+    field valued matrices: ''
+    elementary operations on matrices' rows: ''
+    elementary operations on matrices' columns: ''
+    determinant: ''
+    invertibility: ''
+    rank of a matrice: ''
+    matrix representation of a linear map: ''
+    coordinates change: ''
+    gauss pivot: ''
+    row-reduced matrices: ''
+  endmorphism invariant subspaces: ''
```

Comment:
There seem to be more missing items, and the indentation from here doesn't match the outline.

Reply:
I'll double check this.

Reply:
Page 1 is fine, I purposefully ignored:
> Applications à la résolution desystèmes d’équations linéaires, au calcul de déterminants, à l’inversion des matrices carrées, à ladétermination du rang d’une matrice, à la détermination d’équations définissant un sous-espacevectoriel.

Because those are not really mathematical statements.
They just explain how you perform computations using those concepts.
So really the mathematical statement is in the fact that the determinant gives you the product of eigenvalues, or the trace, the sum of eigenvalues for example.

I'm checking the rest.

Reply:
Okay, the whole 4 should be about bilinear & quadratic forms. I'm writing it.

Reply:
I added a first draft of this

Reply:
Ah, OK. Thanks for the explanation. I think there could be some value in leaving them in, since we will want to build up a nice user interface for performing such computations in Lean as well. Certainly lots of basic lemmas around those and maybe some tactics.

Reply:
Do you mind adding translations of these items (and any others that are skipped) for the reason I laid out above?

If you address this and Johan's comment below re: principal ideal domain, I think we can merge this PR.

Reply:
I would really like this to be merged. The fact that this PR is stuck in a fork that nobody but Ryan can access is very counter-productive. 

I agree that every section that is currently missing will be missing forever since nobody will go back to check, especially since we will probably gradually loose connection to the initial list through refactoring of this list.

But I'm not convinced by Bryan's argument "I think there could be some value in leaving them in, we will want to build up a nice user interface for performing such computations in Lean as well". This is somehow orthogonal to what this list is meant to do. We should rather work on files like https://leanprover-community.github.io/theories/linear_algebra.html if we want to add information about how to manipulate objects from a given corner of the library.

Reply:
OK, I'm willing to cede on this point. I don't have as clear of an idea how strict the translation should be, so do you mind checking over the list one last time and making sure that what's present now is in line with your vision?

Reply:
Sorry, I missed that request. I'll merge this and fix some small things tomorrow. The current process is not fast enough.